src/backend/api/backend.did

```diff
@@ -203,7 +207,7 @@ service : {
   create_my_user_profile : () -> (CreateMyUserProfileResponse);
   update_my_user_profile : (UpdateMyUserProfileRequest) -> (UpdateMyUserProfileResponse);
   update_user_profile : (UpdateUserProfileRequest) -> (UpdateUserProfileResponse);
-  list_proposals : () -> (ListProposalsResponse) query;
+  list_proposals : (opt ListProposalsRequest) -> (ListProposalsResponse) query;
```

Comment:
Shouldn't we keep it non-optional like in all the other methods? Yes, making the request becomes a bit ugly when the filter is empty with `record {}`, but at least that's consistent with the other methods. I'm also fine in leaving it optional and adopt the same logic for the future methods

Reply:
I made it non-optional. At some point in the future we'll have pagination and multiple nervous systems so I think we'll have some mandatory properties then anyway.